docs/evm_compatibility/overview.md

```diff
@@ -40,24 +40,39 @@ Interoperability between Solana and Ethereum EVMs does require certain adaptatio
 ### Precompiles
 Neon supports all precompiled contracts defined on [evm.code](https://www.evm.codes/precompiled?fork=merge) that provide more advanced functionalities, with [certain limitations](./precompiles#limitations) on some precompiled contracts on Neon EVM.
 
+
+Neon also supports native precompiled contracts that are available to our users.
+
+
 ### Gas calculation
-The mechanism of gas consumption and calculation of gas fees on Neon EVM differ from Ethereum. Gas fees on Neon EVM are much cheaper than on Ethereum. 
+The mechanism of gas consumption and calculation of gas fees on Neon EVM differ from Ethereum. Gas fees on Neon EVM are much cheaper than on Ethereum because Solana is the settlement layer. 
+
+<!-- Oleg could provide metrics on one transfer. Once mainnet launched can we compare Neon + Solana to L2 and Rollups Yuri has for tx such as transfers and swaps -->
 
 > Learn more about the [NEON token and how gas fees work on Neon EVM](../../docs/tokens/gas_fees.md).
 
 Several Solana-specific differences also impact smart contract development.
 
-### Upper limit on number of Accounts
-Neon EVM uses Solana Transaction V0 with an extension via [account lookup tables](https://docs.solana.com/developing/lookup-tables): this limits the maximum number of accounts to 64 in a single transaction. Solana requires that all accounts used in a transaction be specified in order to ensure parallel execution of transactions. 
+### Upper limit on number of accounts
+<!-- link to solana tx renamed and relinked -->
+Neon EVM uses [Solana Transaction V0](https://docs.solana.com/developing/versioned-transactions): limiting the maximum number of accounts used in a single transaction to 64. Solana requires that all accounts used in a transaction be specified in order; to ensure parallel execution of transactions.
+
+<!-- go deeper on HOW to modify the contract to constrain account numbers Anton will pass in slack
```

Comment:
@afalaleev We have a note to provide boilerplate code to constrain account numbers

Reply:
@anton-lisanin -- sorry this is for you, not Andrey!

Reply:
some boilerplate code in place -- closing and relocating comments to there